src/main/java/de/hysky/skyblocker/skyblock/dwarven/PickobulusHelper.java

```diff
@@ -0,0 +1,314 @@
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.PRISMARINE_BRICKS)) {
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.DARK_PRISMARINE)) {
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.LIGHT_BLUE_WOOL)) {
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 5;
+		} else if (state.isOf(Blocks.GRAY_WOOL)) {
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()]++;
+		} else if (state.isOf(Blocks.CYAN_TERRACOTTA)) {
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()]++;
+		}
+		handleBreakable(pos, i, j, k);
+	}
+
+	private static void handleGlaciteTunnels(BlockPos pos, BlockState state, int i, int j, int k) {
+		if (state.isOf(Blocks.PACKED_ICE)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()] += 2;
+			drops[MiningDrop.ICE.ordinal()]++;
+			blocks[i][j][k] = Blocks.AIR.getDefaultState();
+		} else if (STAINED_GLASS_BLOCKS.contains(state.getBlock())) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()] += 2;
+			drops[MiningDrop.GEMSTONES.ordinal()]++;
+			blocks[i][j][k] = Blocks.AIR.getDefaultState();
+		} else if (state.isOf(Blocks.POLISHED_DIORITE)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()] += 4;
+			drops[MiningDrop.TITANIUM.ordinal()]++;
+		} else if (state.isOf(Blocks.INFESTED_STONE)) {
+			drops[MiningDrop.HARDSTONE.ordinal()]++;
+		} else if (state.isOf(Blocks.LIGHT_GRAY_CARPET)) {
+			drops[MiningDrop.HARDSTONE.ordinal()]++;
+		} else if (state.isOf(Blocks.PRISMARINE)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.PRISMARINE_BRICKS)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.DARK_PRISMARINE)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 3;
+		} else if (state.isOf(Blocks.LIGHT_BLUE_WOOL)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()] += 5;
+		} else if (state.isOf(Blocks.GRAY_WOOL)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()]++;
+		} else if (state.isOf(Blocks.CYAN_TERRACOTTA)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()]++;
+			drops[MiningDrop.MITHRIL.ordinal()]++;
+			drops[MiningDrop.MITHRIL_POWDER.ordinal()]++;
+		} else if (state.isOf(Blocks.BROWN_TERRACOTTA)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()] += 2;
+			drops[MiningDrop.UMBER.ordinal()]++;
+		} else if (state.isOf(Blocks.TERRACOTTA)) {
+			drops[MiningDrop.MINESHAFT_PITY.ordinal()] += 2;
+			drops[MiningDrop.UMBER.ordinal()]++;
+		} else if (state.isOf(Blocks.RED_SANDSTONE_SLAB)) {
```

Comment:
I didn't see any red sandstone slab?
```suggestion
		} else if (state.isOf(Blocks.SMOOTH_RED_SANDSTONE)) {
```